cobra2snooty_test.go

```diff
@@ -71,7 +71,10 @@ func Echo() *cobra.Command {
 		Aliases: []string{"say"},
 		Short:   "Echo anything to the screen",
 		Long:    "an utterly useless command for testing",
-		Example: "Just run root echo",
+		Example: map[string]string{
```

Comment:
example must be a single string, go is statically typed so you can't do this
```suggestion
		Example: "# Example with intro text\natlas command no intro text"
```